setup.py

```diff
@@ -61,11 +61,9 @@ def get_version(file_name: str, version_variable: str = "__version__") -> str:
     license="MIT",
     install_requires=[
         "GitPython",
-        "tensorstore",
+        "tensorstore >= 0.1.14",
         "file-or-name",
         "importlib_metadata",
```

Comment:
With a bump to 3.7, we can remove the `importlib_metadata` backport too

Reply:
I think `importlib.metadata` was added to stdlib in 3.8 and was provisional until 3.10 based on https://docs.python.org/3/library/importlib.metadata.html and that's the way we seem to be treating it in `git_theta/updates/base.py`

Reply:
You right I commented before I checked lol.